addon/components/frost-tabs.js

```diff
@@ -9,6 +9,7 @@ export default Component.extend(PropTypesMixin, {
   layout: layout,
   classNames: ['frost-tabs'],
   classNameBindings: ['design'],
+  tabComponent: 'frost-tab',
```

Comment:
Can you put this in `propTypes` and use `getDefaultProps ()` to set the default.

Also need to reflect new parameter in https://github.com/ciena-frost/ember-frost-tabs#api